package/src/renderer/components/shapes/Path.tsx

```diff
@@ -22,8 +24,8 @@ export interface PathProps extends CustomPaintProps {
 export const Path = (props: AnimatedProps<PathProps>) => {
   const onDraw = useDrawing(
     props,
-    ({ canvas, paint }, { start, end, stroke, path: pathDef }) => {
-      const path = processPath(pathDef);
+    ({ canvas, paint }, { start, end, stroke, ...pathProps }) => {
+      const path = processPath(pathProps.path).copy();
```

Comment:
Why do we do copy here?

Reply:
trim() is mutating the path.

Reply:
the trim is not used in processPath rather in this component (in the next step)